lib/models/user.js

```diff
@@ -26,10 +26,10 @@ module.exports = function (sequelize, DataTypes) {
       type: DataTypes.TEXT
     },
     accessToken: {
-      type: DataTypes.STRING
+      type: DataTypes.TEXT
```

Comment:
Any reason why this change is needed?

Reply:
Sorry, I should have mentioned this. Our OAuth provider sometimes produces tokens that are longer that 255 chars (which I believe is the maximum STRING length). From [what I understood](https://stackoverflow.com/a/16365828/682095) there isn't a maximum lenght we can assume will be respected.